owncloudApp/src/main/java/com/owncloud/android/presentation/ui/authentication/LoginActivity.kt

```diff
@@ -347,26 +367,79 @@ class LoginActivity : AppCompatActivity(), SslUntrustedCertDialog.OnSslUntrusted
 
     /**
      * OAuth step 1: Get authorization code
+     * Firstly, try the OAuth authorization with Open Id Connect, checking whether there's an available .well-known url
+     * to use or not
      */
-    private fun startOAuthorization() {
+    private fun startOIDCOauthorization() {
         server_status_text.run {
             setCompoundDrawablesWithIntrinsicBounds(R.drawable.progress_small, 0, 0, 0)
             text = resources.getString(R.string.oauth_login_connection)
```

Comment:
I think we should have different messages for oauth and oidc, what do you think?

Reply:
I thought about and I kept it simple, OIDC is OAuth as well so there's nothing weird on using connecting to OAuth server, as well as we do not know if the server we are connecting to is OIDC or not at this point, we just have a bearer authentication header and it could be normal OAuth or OIDC till moving a bit forward, when we say **oidcSupported = true**

Reply:
A question related with oidcSupported, how is it handled with configuration changes?

Reply:
Well, oidcSupported is checked just after setting its value to true, as soon as the user press the log in button, it's a matter of milliseconds I would say, if you are able to tap the button and rotate the device in that time... I wasn't , maybe @jesmrec could manage to break it that way, let's see in QA stage